app/models/user.rb

```diff
@@ -1,5 +1,7 @@
 # frozen_string_literal: true
 
+require 'pdf-reader'
```

Comment:
We don't need this on the User model, only in the class that actually reads the PDF.